### DISCUSSION TITLE
Two Weeks After Approval for the Student Developer Pack, I Randomly Lost All Access to It.  Please Help!

### DISCUSSION THREAD

**Discussion by AdrianeNS** [community/community#123342] [GitHub Education]
### Hi everyone,

On 4/26 I applied for the GitHub Education for the Student Developer Pack. Initially, I had a really hard time getting the website to accept my documents that prove my student status. First, I uploaded a copy of my student ID which includes both the name of my school and the expiration date, but received a message that my ID could not be used because it did not contain the name of my school. Of course, this is not correct as my school's name is clearly printed across the bottom of my card. Then, I uploaded a copy of my invoice for my classes this semester and the website would not accept this at all, and never even gave a reason why.  It just would not go through. So finally, I uploaded a copy of my transcript and again the website said that it couldn't be accepted either because it did not include my school's name. However that is incorrect as well, my school's name is clearly printed on the transcript, along with a date. This alone was an insanely frustrating process but somehow, I eventually got a message that my application was approved. Several days later, I was able to access the Student Developer Pack and Copilot Chat, which was a huge relief.

Then today, out of nowhere, I started having trouble with Copilot GPT in VS Code. It stopped working. This ultimately led to my discovery that my account had been flagged and my student status just disappeared out of nowhere, even though I was approved for the education program for one year. I'm extremely frustrated because I depend on Copilot Chat in VS Code to help me with my schoolwork and now that I suddenly lost access to it, I cannot finish it.  I was able to get my account restored pretty quickly after appealing, but I was never given any reason as to why this happened in the first place.  No email or anything was sent.  Although my account was restored, my student status has simply disappeared, for no apparent reason.  I was already approved and had submitted multiple documents with all of the requirements listed.  There is just no good reason why I should not be able to access my Student Developer Pack.

I have deadlines to turn in my work and this has caused a massive inconvenience which could potentially affect my grades if it's not fixed immediately. I am attaching a copy of the approval email I received from GitHub so you can see that I was already approved for the program a couple weeks ago. I have no idea why my access to my Student Developer Pack and GitHub Copilot just disappeared. I have certainly not done anything wrong. I only have a handful of GitHub Repositories and there is nothing odd or malicious about them. If someone could please fix this asap I would really appreciate it. Please let me know if you have any questions. Thank you.
[GitHub Student Approval.pdf](https://github.com/community/community/files/15280722/GitHub.Student.Approval.pdf)

**Comment by AdrianeNS** [comment]
Is no someone available to fix this issue please?

**Comment by IgorD-lab** [comment]
Contact GitHub Support if you havent already. You can use the free trial untill they respond.

**Comment by AdrianeNS** [comment]
Thanks, I did contact support.  I can't use the free trial because when my Education account was approved, they cancelled my free trial!  I'm so frustrated!

**Reply by IgorD-lab** [reply]
💀
